### PR TITLE
Added links to readme.md and fixed other stuff.

### PR DESCRIPTION
**Pull request by ruth-mapr**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ docker run --net=host --name='myriad-resourcemanager' -t \
 
 If you already had a working Vagrant instance, you will need to run `vagrant reload` in order to allow zookeeper and hdfs port-forwarding.
 
-Still having problems connecting to Zookeeper? Be sure that your zkServer and MesosMaster values are correct in the [myriad-config-default.yml](https://github.com/mesos/myriad/blob/phase1/myriad-scheduler/src/main/resources/myriad-config-default.yml) file.
+Still having problems connecting to Zookeeper? Be sure that your zkServer and MesosMaster values are correct in the [myriad-config-default.yml](../myriad-scheduler/src/main/resources/myriad-config-default.yml) file.
```
Comment (DarinJ) [inline docker/README.md]: Nit, I would change this to:
If you are still having problems connecting to Zookeeper, be sure ...

Comment (DarinJ) [inline docker/README.md]: Also, in general I place myriad-config-default.yml in $HADOOP_HOME/etc/hadoop/, this puts it in the classpath before the one packaged in `srv/main/resources/`, and allows for configuration without recompilation.  I think this should be the preferred method and the documentation should reflect that.

**Comment by kensipe** [comment]
@ruth-mapr were you going to respond to @DarinJ comments?

**Comment by ruth-mapr** [comment]
I really didn't work on this docker/readme.md file and I'm not familiar with docker. But I can make the changes.

**Comment by ruth-mapr** [comment]
oops

**Comment by ruth-mapr** [comment]
Made the 3 changes: 1. yml file link correction, 2. Grammar re: connecting to Zookeeper, 3. Sentence about placing the file in $HADOOP_HOME/etc/hadoop/.

**Comment by jpgilaberte** [comment]
This PR is old but looks good. I have reviewed it and it is necessary to correct:
- README.md, [Sample: yarn-site.xml] (sample-yarn-site.md) by [Sample: yarn-site.xml] (docs / sample-yarn-site.md)
- docs / vagrant.md, there are two characters that can be removed from the Mesos Master example.

It would be necessary to resolve the conflicts also before merging it. I do not have access, I would have to fix it from my repo and make a new PR by keeping a comment to the original author.

It would be correct??

**Comment by javiroman** [comment]
May you create an new PR with the changes? We can close this PR and add you and @ruth-mapr as co-authors.

**Comment by javiroman** [comment]
@jpgilaberte Anyway let @ruth-mapr update the PR whether she have time. Let's wait for a few days to let her do it.

**Pull request closed without merging**